vnext/stubs/double-conversion/double-conversion.h

```diff
@@ -7,6 +7,7 @@
 #pragma once
 
 #include <glog/logging.h>
+#include <charconv>
```

Comment:
I would expect code formatting to move this `#include` to the first position.

Reply:
looks like clang format chose to put it here :)